ipld/read.go

```diff
@@ -11,9 +11,9 @@ import (
 
 	"github.com/celestiaorg/rsmt2d"
 
-	"github.com/celestiaorg/celestia-core/pkg/da"
-	"github.com/celestiaorg/celestia-core/pkg/wrapper"
 	"github.com/celestiaorg/celestia-node/ipld/plugin"
+	"github.com/tendermint/tendermint/pkg/da"
+	"github.com/tendermint/tendermint/pkg/wrapper"
```

Comment:
Hmm. Now I have a concern about importing Tendermint directly because it does not have those packages and likely will never do. 

I feel like this is confusing. It kinda lies that Tendermint has those packages but it is not. For any newcomer who decides to look at the sources, this may end up as a road to nowhere.

Unfortunately, Tendermint is not a black box to do so and now I don't like this idea :(

Reply:
Yeah I agree this is kind of confusing :/

Reply:
I agree that it is confusing, and as discussed a while ago somewhere we should eventually move those packages to an external repo to avoid this exact thing. Until then, thanks to go modules we have no other option unless we wish change how celestia-core declare's it package (and therefore re-change all the import everywhere and make pulling changes from upstream feel like pulling teeth).

Reply:
Okey :(

Reply:
Okey ;(